crates/iota-indexer/src/indexer_reader.rs

```diff
@@ -1990,6 +1992,17 @@ impl IndexerReader {
         .await
         .map_err(Into::into)
     }
+
+    /// Get the participation metrics. Participation is defined as the total
+    /// number of unique addresses that have delegated stake in the current
+    /// epoch. Includes both staked and timelocked staked IOTA.
+    pub fn get_participation_metrics(&self) -> IndexerResult<ParticipationMetrics> {
+        let stored_participation_metrics = run_query!(&self.pool, |conn| {
+            diesel::sql_query("SELECT * FROM participation_metrics")
+                .get_result::<StoredParticipationMetrics>(conn)
+        })?;
+        Ok(stored_participation_metrics.into())
```

Comment:
Nit: I suppose we could also shorten the function impl here.

```suggestion
        run_query!(&self.pool, |conn| {
            diesel::sql_query("SELECT * FROM participation_metrics")
                .get_result::<StoredParticipationMetrics>(conn)
        })
        .map(Into::into)
```